### PR TITLE
Add proc_creation_win_susp_msbuild

### PR DESCRIPTION
**Pull request by frack113**
Detects  'Msbuild.exe' run  by a uncommun parent

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_susp_msbuild.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+title: Suspicious Msbuild Usage
+id: 33be4333-2c6b-44f4-ae28-102cdbde0a31
+status: experimental
+description: Detects suspicious use of 'Msbuild.exe' by a uncommun parent
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_msbuild.yml:4]:
```suggestion
description: Detects suspicious execution of 'Msbuild.exe' by a uncommon parent process
```

File: rules/windows/process_creation/proc_creation_win_susp_msbuild.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+title: Suspicious Msbuild Usage
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_msbuild.yml:1]:
```suggestion
title: Suspicious Msbuild Execution By Uncommon Parent Process
```

File: rules/windows/process_creation/proc_creation_win_susp_msbuild.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+date: 2022/11/17
+tags:
+    - attack.defense_evasion
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        - Image|endswith: '\MSBuild.exe'
+        - OriginalName: 'MSBuild.exe'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_msbuild.yml:18]:
```suggestion
        - OriginalFileName: 'MSBuild.exe'
```

File: rules/windows/process_creation/proc_creation_win_susp_msbuild.yml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    filter_parent:
+        ParentImage|endswith:
+            - '\devenv.exe'
+            - '\cmd.exe'
+            - '\msbuild.exe'
+            - '\python.exe'
+            - '\explorer.exe'
+            - 'nuget.exe'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_susp_msbuild.yml:26]:
```suggestion
            - '\nuget.exe'
```

Comment (frack113) [reply rules/windows/process_creation/proc_creation_win_susp_msbuild.yml:26]: I ahve miss this one

**Pull request merged**